app/code/ElTrubetskaia/AskQuestion/view/frontend/templates/product/view/question_list.phtml

```diff
@@ -0,0 +1,14 @@
+<?php /** @var \ElTrubetskaia\AskQuestion\Block\Question $block */
+$askQuestions = $block->getAskQuestions();
+?>
+<?php if ($askQuestions !== null) : ?>
+    <ul>
+        <?php /** @var \ElTrubetskaia\AskQuestion\Model\AskQuestion $askQuestion */
+        foreach ($askQuestions as $askQuestion) :
+            ?>
+            <?php if ($askQuestion->getSku() === $block->getCurrentProduct()->getSku()) : ?>
```

Comment:
This condition should be checked on database side. Please create \ElTrubetskaia\AskQuestion\Block\Question::getAskQuestionsForProduct(...)
and pass it to collection (addFieldToFilter)

Reply:
fixed. thanks!